src/main/java/org/ibp/api/java/impl/middleware/common/validator/GermplasmUpdateDtoValidator.java

```diff
@@ -194,6 +195,15 @@ protected void validateAttributeAndNameCodes(final BindingResult errors, final S
 		}
 	}
 
+	protected void validateProgenitorMustNotBeGid(final BindingResult errors, final List<GermplasmUpdateDTO> germplasmUpdateDTOList) {
```

Comment:
@cheryleigh Please use a more self explanatory name for the function
Does this work when excel file contains GUID insted of GID ?